fxevent/zap.go

```diff
@@ -24,145 +24,175 @@ import (
 	"strings"
 
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
 // ZapLogger is an Fx event logger that logs events to Zap.
 type ZapLogger struct {
 	Logger *zap.Logger
+
+	logLevel   *zapcore.Level
```

Comment:
since info is the zero value, i would drop the pointer on this one.
```suggestion
	logLevel   zapcore.Level  // default: info
```